src/selection.js

```diff
@@ -13,16 +13,18 @@ export const findParentNode = predicate => selection => {
   for (let i = $from.depth; i > 0; i--) {
     const node = $from.node(i);
     if (predicate(node)) {
+      const start = $from.start(i);
       return {
-        pos: $from.start(i),
+        pos: start - 1,
```

Comment:
I probably would just rely on PM to tell you what the position at the node before is using `$from.before(i)` rather than assuming it's `-1` (even though it should always be, since you'll only compare against parent, and therefore non-leaf nodes)

Reply:
agreed!

Reply:
the only reason I did that is because on depth 0 it throws an error because there's nothing "before". I'll add a check for that.